<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBoxCommon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>Common</string>
   </property>
   <widget class="QLabel" name="labelOutputformat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Output format</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxOutputFormat">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>20</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>mp3 (320 kbps)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>wav (uncompressed)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>flac (lossless)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>m4a (lossy)</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBoxIsDemo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelIsDemo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Put on demo page</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSecert">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>400</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Secert</string>
   </property>
   <widget class="QLabel" name="labelAPI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>API token</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAPI">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
